ios/fastlane/Fastfile

```diff
@@ -12,19 +12,130 @@
 
 default_platform(:ios)
 
+
+def get_changelog()
+  
+  Dir.chdir("..") do
+
+    branch_name = git_branch
+    commit = last_git_commit
+    
+    matched_env_msg = nil
+    ci_matched_env = ENV["CI_MATCHED_ENV"]
+    
+    if ci_matched_env != nil && !ci_matched_env.empty?
+      matched_env_msg = " using \"#{ci_matched_env}\" environment"
+    end
+
+    changelog_msg = "(automatically built from #{commit[:abbreviated_commit_hash]} on #{branch_name}#{matched_env_msg})"
```

Comment:
This is currently by design as there's no branch name available for pull requests (GitHub Actions checks out the repo as a specific merge ref not branch). The build workflow is only intended to be run from commits (not PRs) in which case this should work.

Checkout https://github.com/KenthLjungHBG/CI-CD-Test/runs/2517310318 for a test run example of it working (CTRL+F for "changelog").

The workflow can only be manually ran/tested from specific branches once the workflow file exists on the default branch (i.e. after this PR is merged), so at the moment it can only be tested as a "pull_request"-triggered workflow.